cmd/root.go

```diff
@@ -208,6 +231,48 @@ func getImageForName(imageName string) (pkgutil.Image, error) {
 	}, nil
 }
 
+func getExtractPathForImage(imageName string, image v1.Image) (string, error) {
+	if isDaemon {
```

Comment:
Can we check if something is from the daemon here, instead of at the package level? I'm still confused about how this would work with two images, one from a daemon and one from a registry.

Reply:
Picking up the latest changes to go-containerregistry eliminates the need for this, removed.